### PR TITLE
Add support for merge queues to the CI setup

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

File: .github/workflows/vast.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ on:
     types:
       - opened
       - synchronize
+  merge_group:
+    types:
+      - checks_requested
```
Comment (dominiklohmann) [inline .github/workflows/vast.yaml:20]: This also needs to be done for the analysis, blocked, and style-check workflow files.

**Review by dominiklohmann — APPROVED** [review]
Reading through the docs I think this is a safe change to make. We did discuss in Slack how to handle the rev count in the version.json file, which I anticipate to clash a bit with merge queues. But we can fix that in a follow-up PR.

File: .github/workflows/vast.yaml (hunk 1 above)
Comment (dominiklohmann) [inline .github/workflows/vast.yaml:20]: Below this we have the concurrency.cancel_in_progress section. We should consider also enabling that for merge queues.

Comment (dominiklohmann) [reply .github/workflows/vast.yaml:20]: We talked about this quickly in the office, and agreed that it's just style check we need to add. I misunderstood the trigger.

Comment (tobim) [reply .github/workflows/vast.yaml]: We agreed that `Tenzir` and `Style Check` are all we need.

**Review by dominiklohmann — APPROVED** [review]

**Pull request merged**